packages/signers/src/turnkey-secp256k1-signer.ts

```diff
@@ -0,0 +1,124 @@
+import { TurnkeyClient } from "@turnkey/http";
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
+import { ethers } from "ethers"; // v6 helpers
+import 'dotenv/config';
```

Comment:
The env vars wont always be defined in a .env file, we shouldn't control loading env vars in library code